src/tokens/slap.ts

```diff
@@ -0,0 +1,25 @@
+import { defaultFetcherOptions, SupplyFetcher } from "../types";
+import { getAmountInAddresses, getBlockFrostInstance } from "../utils";
+
+const SLAP = "1e2ab14b6921c545b836525477921fe000e14a3502fa729f60e8de85534c4150";
+
+const fetcher: SupplyFetcher = async (options = defaultFetcherOptions) => {
+  const blockFrost = getBlockFrostInstance(options);
+  const total = 100_000_000_000;
+  const treasuryRaw = await getAmountInAddresses(blockFrost, SLAP, [
+    "stake1ux366xgzh78hh8hk0ep0824pz7ha306uese7u644h7wn86quca6l3", // $slapventures
+  ]);
+
+  const burnRaw = await getAmountInAddresses(blockFrost, SLAP, [
+    "addr1w8qmxkacjdffxah0l3qg8hq2pmvs58q8lcy42zy9kda2ylc6dy5r4", //$burnsnek
```

Comment:
what? this address doesn't have SLAP token

Reply:
It does have it :-)  https://cardanoscan.io/tokenHoldings/addr1w8qmxkacjdffxah0l3qg8hq2pmvs58q8lcy42zy9kda2ylc6dy5r4
![Zrzut ekranu 2024-01-01 215340](https://github.com/minswap/market-cap/assets/63907980/7536f9cf-d85d-4c5d-9621-c403d8f9543b)

Reply:
but if you check on Cardanoscan directly it actually doesn't find SLAP until you "expand" tokens list. Maybe it's a Cardanoscan error due to too many tokens on the wallet? No idea. But the tokens are 100% there - this is the transaction from 18th Dec: https://cardanoscan.io/transaction/667394df5e528e8f8268363a1580aff4ae06f1b6efee7fafec460603990973b7